test/narrative_ref_test.py

```diff
@@ -1,63 +1,63 @@
 import unittest
 
+import pytest
 from StaticNarrative.narrative_ref import NarrativeRef
 
+WSID = 123
+OBJID = 456
+VER = 7
 
 class NarrativeRefTestCase(unittest.TestCase):
-    def test_ok(self):
-        ref = NarrativeRef({"wsid": 123, "objid": 456, "ver": 7})
-        self.assertEqual(ref.wsid, 123)
-        self.assertEqual(ref.objid, 456)
-        self.assertEqual(ref.ver, 7)
+    def test_ok(self) -> None:
+        ref = NarrativeRef({"wsid": WSID, "objid": OBJID, "ver": VER})
+        assert ref.wsid == WSID
+        assert ref.objid == OBJID
+        assert ref.ver == VER
```

Comment:
convert this file over to a more pytest-y form -- aim to switch over to pytest completely as part of this work (but not this PR)